src/backend/TrafficCourts/Citizen.Service/Services/Impl/CitizenDocumentService.cs

```diff
@@ -46,7 +46,13 @@ public async Task DeleteFileAsync(Guid fileId, CancellationToken cancellationTok
 
         FileSearchResult file = searchResults[0];
 
-        DocumentProperties properties = new DocumentProperties(file.Metadata, file.Tags);
+        DocumentProperties properties = new(file.Metadata, file.Tags);
+        
+        // Citizen Portal should only have access to Citizen documents
+        if (properties.DocumentSource is not null && properties.DocumentSource is not DocumentSource.Citizen) {

```

Comment:
Same

Reply:
Same as above. I do mean Citizen here. The document to delete should only be a citizen-uploaded document, throw an error otherwise.